### ISSUE TITLE
Make it possible to add classes to a ColumnGroup

### ISSUE THREAD

**Issue by rmilesson** [facebook/fixed-data-table#203]
I would very much like the capabilities to add classes to a ColumnGroup in a neat way, just like it's possible to use the cellClassName/headerClassName-properties to target data cells and headers respectively.

I could most likely just implement it myself and do a PR, but I'm just throwing it out here to see what people think about it before I do some work on it.

**Comment by ehzhang** [comment]
Certainly! This is actually one of the reasons why we're moving into using cell 'templates' instead of renderers and getters in the next release. That way, you can assign your own classNames for your cells without needing an API to do so. This will likely be included as a part of the release #193 .

**Comment by rmilesson** [comment]
Alright, yes, that would be nice.

However, I just fiddled about a bit, if I pass a `headerClassName` to a ColumnGroup, it will be applied to it, but not carry on downwards to the columns. So with a bit of magic I got it to work just as fine on our end for now.
